cdap-app-fabric/src/main/java/io/cdap/cdap/internal/sysapp/SystemAppEnableExecutor.java

```diff
@@ -51,50 +54,55 @@
 public class SystemAppEnableExecutor {
 
   private static final Logger LOG = LoggerFactory.getLogger(SystemAppEnableExecutor.class);
+  private static final Logger LIMITED_LOGGER = Loggers
+      .sampling(LOG, LogSamplers.limitRate(TimeUnit.SECONDS.toMillis(100)));
+
   private static final Gson GSON = new Gson();
   private final ApplicationLifecycleService appLifecycleService;
   private final ProgramLifecycleService programLifecycleService;
 
   @Inject
   SystemAppEnableExecutor(ApplicationLifecycleService appLifecycleService,
-                          ProgramLifecycleService programLifecycleService) {
+      ProgramLifecycleService programLifecycleService) {
```

Comment:
indentation

Reply:
same comment for the rest of the changes, earlier fixes look like they were undone.